src/App.tsx

```diff
@@ -0,0 +1,23 @@
+import './App.scss'
+import ProductDetails from './components/products/product-details/product-details'
+import ProductList from './components/products/product-list/product-list'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import ShoppingCart from './components/cart/shopping-cart/shopping-cart'
+
+function App() {
+  return (
+    <>
+    <Routes>
+      <Route path='/' element={<Navigate to='/products'/>} />
+      <Route path='/products'>
```

Comment:
Maybe `<Outlet />` can be used to specify that nested routes will be rendered within `<Route path="/products" />`